README.md

```diff
@@ -1,63 +1,200 @@
+
+1. "(I've Had) The Time of My Life"
+2. "Be My Baby"
+3. "She's Like the Wind"
+4. "Hungry Eyes"
+5. "STAY"
+6. "Yes"
+7. "You Don't Own Me"
+8. "Hey Baby"
+9. "Overload"
+10. "Love Is Strange"
+11. "Where Are You Tonight?"
+12. "In the Still of the Night"
+
+Did you create that property? If you need some help, here's what `TableViewController`'s class definition should look like so far:
+
+```swift
+class TableViewController: UITableViewController {
+    let dirtyDancingSoundtrack = [
+        "(I've Had) The Time of My Life",
+        "Be My Baby",
+        "She's Like the Wind",
+        "Hungry Eyes",
+        "STAY",
+        "Yes",
+        "You Don't Own Me",
+        "Hey Baby",
+        "Overload",
+        "Love Is Strange",
+        "Where Are You Tonight?",
+        "In the Still of the Night"
+    ]
+}
+```
+
+(Note that there are a few ways to create properties in Swift. If your `dirtyDancingSoundtrack` property is defined a bit differently, that's okay!)
+
+Now you can fill in the methods that will provide data back to your table view. Do you remember the three methods you should provide so the table view can be drawn correctly?
+
+The first is `numberOfSectionsInTableView(_:)`, which returns the number of _sections_ in the table view. In this relatively simple app, there is only one section, so this method can return 1:
+
+```swift
+override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    return 1
+}
+```
+
+The second method you must provide is `tableView(_:numberOfRowsInSection:)`, which returns the number of _rows_ for a given section. There's only one section, so you don't have to worry about the section in this method. There is one row for each song, so you can just return the total number of songs:
+
+```swift
+override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return dirtyDancingSoundtrack.count
+}
+```
+
+Finally, the trickiest of the three methods: `tableView(_:cellForRowAtIndexPath:)`. Remember, this method passes in an exact section (which is always `0` in this app, since there's only one section) and row number, and is responsible for returning a rendered _cell_ corresponding to that row. This method is a bit tricky because you have to worry about _dequeuing_ a cell (do you remember why from the previous lesson) and setting its text label correctly.
+
+Go ahead and try this one yourself. Refer back to the previous lesson if you need to.
+
+When you've implemented `tableView(_:cellForRowAtIndexPath:)`, it should look something like this:
+
+```swift
+override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCellWithIdentifier(reuseIdentifier, forIndexPath: indexPath)
```

Comment:
We might have to add at the top for them to create this instance property called `reuiseIdentifier`, assigning it a value of "BasicCell" above? Or maybe... have them do it here at this point. I don't think we told them to do this so they might think they should have access to this `reuiseIdentifier` instance property that doesn't exist.